### PR TITLE
Onchain information error

### PR DESCRIPTION
**Pull request by KillariDev**
fix https://github.com/DarkFlorist/TheInterceptor/issues/638
image

### CODE REVIEW

File: app/ts/components/pages/AddNewAddress.tsx [hunk 1]
```diff
@@ -330,6 +331,28 @@ export function AddNewAddress(param: AddAddressParam) {
 		})
 	}
 
+	function showOnChainVerificationErrorBox() {
+		return incompleteAddressBookEntry.entrySource === 'OnChain' && (incompleteAddressBookEntry.type === 'ERC20' || incompleteAddressBookEntry.type === 'ERC721')
+	}
+
+	function isSubmitButtonDisabled() {
+		return !areInputValid()
```
Comment (jubalm) [inline app/ts/components/pages/AddNewAddress.tsx:339]: just minor grammar with `are` implying many. either `isInputValid` or `areInputsValid` for clarity

File: app/ts/components/pages/AddNewAddress.tsx [hunk 2]
```diff
@@ -358,16 +381,24 @@ export function AddNewAddress(param: AddAddressParam) {
...
+				<div style = 'padding-left: 10px; padding-right: 10px; margin-bottom: 10px; height: 80px'>
 					{ errorString === undefined ? <></> : <Notice text = { errorString } /> }
-					{ errorString === undefined && incompleteAddressBookEntry.duplicateStatus === 'Duplicates' ? <Notice text = { 
-						`There already exists ${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type} with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }".`
-					 } /> : <></> }
+					{ errorString === undefined && incompleteAddressBookEntry.duplicateStatus === 'Duplicates' ? <>
+						<Notice text = { `There already exists ${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type} with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }".` } />
```
Comment (jubalm) [inline app/ts/components/pages/AddNewAddress.tsx:387]:
```suggestion
						<Notice text = { `An entry (${ incompleteAddressBookEntry.duplicateEntry.type === 'addressInfo' ? 'an address' : incompleteAddressBookEntry.duplicateEntry.type}) with ${ 'symbol' in incompleteAddressBookEntry.duplicateEntry ? `symbol "${ incompleteAddressBookEntry.duplicateEntry.symbol }" and` : '' } name "${ incompleteAddressBookEntry.duplicateEntry.name }" already exists.` } />
```

**Review by jubalm — APPROVED** [review]

File: app/ts/components/pages/AddNewAddress.tsx (hunk 2 above)
Comment (KillariDev) [reply app/ts/components/pages/AddNewAddress.tsx:387]: I don't think "An entry already exists." is proper english, you need to have verb in it ("there")

File: app/ts/components/pages/AddNewAddress.tsx (hunk 1 above)
Comment (KillariDev) [reply app/ts/components/pages/AddNewAddress.tsx:339]: `input` is also plurar for `input` https://grammarhow.com/input-or-inputs/

**Pull request merged**